core/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+# Build libdronecore
+
+# This finds thread libs on Linux, Mac, and Windows.
+find_package(Threads REQUIRED)
+
+include_directories(
+    SYSTEM ${CMAKE_SOURCE_DIR}/libs/include
+)
+
+if (ANDROID)
+    include_directories(curl-android-ios/prebuilt-with-ssl/android/include)
+endif()
+
+if (IOS)
+    include_directories(curl-android-ios/prebuilt-with-ssl/ios/include)
+endif()
+
+if (IOS OR ANDROID OR MSVC OR APPLE)
+    set(library_type "STATIC")
```

Comment:
@julianoes: I don't know about MSVC, but why not creating shared libraries for the other platforms, as we already do for Linux?